.github/workflows/backup-production.yml

```diff
@@ -0,0 +1,16 @@
+name: production
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
This would run every day at 8pm our time (UTC -4), what do you think about running it our (local) "off" hours? Like 4am or something, so that would be 8am UTC.

Reply:
Great idea! 4am sounds good